### DISCUSSION TITLE
The easiest way to install HippoPlayer on Arch-Linux?

### DISCUSSION THREAD

**Discussion by Kepalu** [emoon/HippoPlayer#199] [Q&A]
Believe it or not, despite the fact that I'm using Arch Linux for many, many years, I'm a very vanilla user and I always depend on AUR. I tried to follow the instructions to install HippoPlayer myself but that doesn't work for me. Would anyone happen to know if there is any simple way to just directly install it without using AUR?

**Comment by emoon** [comment]
Hi,

Currently on Linux the only way to use HippoPlayer is to build the code. There is an AUR for Arch, but it hasn't been updated in a while and doesn't work. There has been some comments on it here https://aur.archlinux.org/packages/hippoplayer-git but the user that did it hasn't updated with the latest way to build it.

Otherwise the build instructions here https://github.com/HippoPlayer/HippoPlayer#linux should work (it's what the CI is using to build the code)

Later on I hope to do a AppImage for HippoPlayer https://appimage.org but I haven't got around to that yet

**Comment by knochenhans** [comment]
Hi there, I posted an updated PKGBUILD for the current release as a comment at [https://aur.archlinux.org/packages/hippoplayer](https://aur.archlinux.org/packages/hippoplayer). The package is unmaintained but I hope this helps for now. The only moving target for coming releases is the pkgver and that sed line changing hippo_core_loader.c (which is necessary for some reason for Arch).

**Reply by emoon** [reply]
Thanks! :)

**Reply by knochenhans** [reply]
I just adopted the package so it hopefully it will be easier to install HippoPlayer for Arch in the future (just need to figure out how AUR works for maintainers ^^)
